.github/scripts/validateWorkflowSchemas.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+##########################################################
+#    Validate GitHub action and workflow yaml schemas    #
+##########################################################
+
+SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+readonly SCRIPT_DIR
+GITHUB_DIR="$(dirname "$SCRIPT_DIR")"
+readonly GITHUB_DIR
```

Comment:
You can simplify similar to above, script_dir is only used once and github_dir throughout